builtin/am.c

```diff
@@ -2284,10 +2284,10 @@ int cmd_am(int argc, const char **argv, const char *prefix)
 			N_("skip the current patch"),
 			RESUME_SKIP),
 		OPT_CMDMODE(0, "abort", &resume.mode,
-			N_("restore the original branch and abort the patching operation."),
+			N_("restore the original branch and abort the patching operation"),
```

Comment:
@adlternative: thanks for doing this cleanup.

You can help your commit message by following the standard pattern. I recommend something like:

```
builtin/*: update usage format

According to the "Error Messages" section of
Documentation/CodingGuidelines, we should not end in a full
stop or start with a capital letter. Fix old error and usage
messages to match this expectation.

(sign-off)
```

Feel free to use any or all of this in your commit message, but be sure the line width makes sense as I didn't check it.

Reply:
ok,I will change it!By the way, I discovered that some of the checks failed, I will check again.